packages/augur-ui/src/modules/auth/components/connect-dropdown/connect-dropdown.tsx

```diff
@@ -156,6 +160,19 @@ const ConnectDropdown = (props: ConnectDropdownProps) => {
           />
         </div>
 
+        {universe.forkingInfo &&
```

Comment:
Set ‘parent’ property to null on universe in reducer, add to type in types.ts. 

Add check for parent here with check for forkingInfo. If parent isnt null then also show button